depends/common/flac/02-relax-linux-OS-detection.patch

```diff
@@ -5,7 +5,7 @@
  
  case "$host" in
 -	*-pc-linux-gnu)
-+	*-linux-*)
++	*-linux-gnu*)
```

Comment:
since this already broke once, I'd recommend being more precise then less.
e.g.
*-pc-linux-gnu || *-linux-gnueabi*

Reply:
will update
just for reference, the first change was taken from upstream, https://git.xiph.org/?p=flac.git;a=commit;h=55721556161e6ab209f940f5023bc44b4051524a but it appeared to not fit our needs.